### PR TITLE
Feature/order api

### PR DESCRIPTION
**Pull request by magdumsuraj07**
Implemented APIs for orders endpoint.
- Getting the order
- Creating an order
- Creating order items
- Updating an order
- Signals

### CODE REVIEW

File: store/serializers.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 from decimal import Decimal
+from django.db import transaction
 from rest_framework import serializers
-from store.models import Cart, CartItem, Customer, Product, Collection, Review
+from .models import Cart, CartItem, Customer, Product, Collection, Review
+from .models import Cart, CartItem, Order, OrderItem, Product, Collection, Review
```
Comment (Avadhut-R-M) [inline store/serializers.py]: @magdumsuraj07 import all models in single line instead of multiple

File: store/views.py [hunk 2]
```diff
@@ -2,22 +2,60 @@
...
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from store.pagination import DefaultPagination
-from store.permissions import FullDjangoModelPermissions, IsAdminOrReadOnly, ViewCustomerHistoryPermission
+from store.permissions import IsAdminOrReadOnly, ViewCustomerHistoryPermission
 from .filters import ProductFilter
 from .models import Cart, CartItem, OrderItem, Product, Collection, Review, Customer
-from .serializers import AddCartItemSerializer, CartItemSerializer, CartSerializer, CollectionSerializer, CustomerSerializer, ProductSerializer, ReviewSerializer, UpdateCartItemSerializer
+from .serializers import AddCartItemSerializer, CartItemSerializer, CartSerializer, CollectionSerializer, CreateOrderSerializer, CustomerSerializer, ProductSerializer, ReviewSerializer, UpdateCartItemSerializer, UpdateOrderSerializer
+from .models import Cart, CartItem, OrderItem, Product, Collection, Review, Order
+from .serializers import AddCartItemSerializer, CartItemSerializer, CartSerializer, CollectionSerializer, OrderSerializer, ProductSerializer, ReviewSerializer, UpdateCartItemSerializer
```
Comment (Avadhut-R-M) [inline store/views.py]: for serialiser import in single line (15,17)

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Avadhut-R-M) [inline .vscode/settings.json:1]: its better if .vscode file is ignored

File: store/views.py (hunk 2 above)
Comment (magdumsuraj07) [reply store/views.py]: Imported in a single line and changed few lines for better formatting.

File: store/serializers.py (hunk 1 above)
Comment (magdumsuraj07) [reply store/serializers.py]: Imported in a single line and changed few lines for better formatting.

File: .vscode/settings.json (hunk 3 above)
Comment (magdumsuraj07) [reply .vscode/settings.json:1]: Removed .vscode from repository

**Review by Avadhut-R-M — APPROVED** [review]

**Pull request merged**